R/packages.R

```diff
@@ -178,7 +185,14 @@ env_download_wasm_core_packages <- function() {
   strsplit(pkgs, "\\s*[ ,\n]\\s*")[[1]]
 }
 
-download_wasm_packages <- function(appdir, destdir, package_cache) {
+download_wasm_packages <- function(appdir, destdir, package_cache, max_filesize) {
+  max_filesize_missing <- Sys.getenv("SHINYLIVE_DEFAULT_MAX_FILESIZE") == "" && is.null(max_filesize)
+  max_filesize_cli_fn <- if (max_filesize_missing) cli::cli_warn else cli::cli_abort
+
+  max_filesize <- max_filesize %||% default_max_filesize()
+  max_filesize <- if (is.na(max_filesize) || (max_filesize < 0)) Inf else max_filesize
+  max_filesize <- fs::fs_bytes(max_filesize)
```

Comment:
Safe guard parsed value

```suggestion
  max_filesize_val <- max_filesize
  max_filesize <- fs::fs_bytes(max_filesize)
  if (is.na(max_filesize)) {
    cli::cli_warn(c(
      "!" = "Could not parse `max_filesize` value: {.code   max_filesize_val }",
      "i" = "Setting to {.code SHINYLIVE_DEFAULT_MAX_FILESIZE }"
    ))
    max_filesize <- fs::fs_bytes(SHINYLIVE_DEFAULT_MAX_FILESIZE)
  }
```

Reply:
Minor tweak made here to use e.g. `{.code {max_filesize_val}}` so that the values are printed by cli.